Convert all helm-template instructions to helm-install (#828)
rancher/rancher-docs: docs/pages-for-subheaders/enable-experimental-features.md

:::

### Enabling Features for Air Gap Installs

> For an air gap installation of Rancher, you need to add a Helm chart repository and download a Helm chart before installing Rancher with Helm. For details, refer to the [air gap installation documentation.](../getting-started/installation-and-upgrade/other-installation-methods/air-gapped-helm-cli-install/install-rancher-ha.md)

Reviewer: ```suggestion

To perform an [air gap installation of Rancher](../getting-started/installation-and-upgrade/other-installation-methods/air-gapped-helm-cli-install/install-rancher-ha.md), add a Helm chart repository and download a Helm chart, then install Rancher with Helm.
```

> Here is an example of a command for passing in the feature flag names when installing the Helm chart. In the below example, two features are enabled by passing the feature flag names in a comma separated list.

Reviewer: ```suggestion
When you install the Helm chart, you should pass in feature flag names in a comma separated list, as in the following example:
```

> The Helm command is as follows:

Reviewer: ```suggestion
```
